schema/ext/vulnerability-1.0.xsd

```diff
@@ -0,0 +1,268 @@
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:enumeration>
+            <xs:enumeration value="Other">
+                <xs:annotation>
+                    <xs:documentation xml:lang="en">
+                        Use this if the risk scoring methodology is not based on any of the options above
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:enumeration>
+        </xs:restriction>
+    </xs:simpleType>
+
+    <xs:complexType name="scoreType">
+        <xs:annotation>
+            <xs:documentation xml:lang="en">
+                Defines the numerical risk score of a vulnerability
+            </xs:documentation>
+        </xs:annotation>
+        <xs:sequence>
+            <xs:element name="score" minOccurs="0" maxOccurs="1">
+                <xs:complexType>
+                    <xs:sequence>
+                        <xs:element name="base" type="xs:decimal" minOccurs="0" maxOccurs="1">
+                            <xs:annotation>
+                                <xs:documentation xml:lang="en">
+                                    The base score of the security vulnerability (Refer CVSS standard for example)
+                                </xs:documentation>
+                            </xs:annotation>
+                        </xs:element>
+                        <xs:element name="impact" type="xs:decimal" minOccurs="0" maxOccurs="1">
+                            <xs:annotation>
+                                <xs:documentation xml:lang="en">
+                                    The impact subscore of the security vulnerability (Refer CVSS standard for example)
+                                </xs:documentation>
+                            </xs:annotation>
+                        </xs:element>
+                        <xs:element name="exploitability" type="xs:decimal" minOccurs="0" maxOccurs="1">
+                            <xs:annotation>
+                                <xs:documentation xml:lang="en">
+                                    The exploitability subscore of the security vulnerability (Refer CVSS standard for
+                                    example)
+                                </xs:documentation>
+                            </xs:annotation>
+                        </xs:element>
+                    </xs:sequence>
+                </xs:complexType>
+            </xs:element>
+            <xs:element name="severity" type="dg:severityType" minOccurs="0" maxOccurs="1"/>
+            <xs:element name="source" type="dg:scoreSourceType" minOccurs="0" maxOccurs="1"/>
+            <xs:element name="vector" type="xs:string" minOccurs="0" maxOccurs="1">
+                <xs:annotation>
+                    <xs:documentation xml:lang="en">
+                        textual representation of the metric values used to score the vulnerability
```

Comment:
minor: change `textual` for `Textual`, looks better, since the other documentation starts with capital letter

Reply:
> minor: change `textual` for `Textual`, looks better, since the other documentation starts with capital letter

👍 5c75f6f